### ISSUE TITLE
bad operand type for unary ~: 'str'

### ISSUE THREAD

**Issue by tjjh89017** [tusharsadhwani/zxpy#5]
Hi 

This is my code for find some zip in dir
but it cause the exception below
```l = (~f'find -maxdepth 1 -name "*{x}*.zip" -type f').splitlines()```

 ``` 
Traceback (most recent call last):
  File "/usr/local/bin/zxpy", line 8, in <module>
    sys.exit(cli())
  File "/usr/local/lib/python3.8/dist-packages/zx.py", line 55, in cli
    run_zxpy(filename, module)
  File "/usr/local/lib/python3.8/dist-packages/zx.py", line 85, in run_zxpy
    exec(compile(module, filename, mode='exec'))
  File "./test.py", line 21, in <module>
    l = (~f'find -maxdepth 1 -name "*{x}*.zip" -type f').splitlines()
TypeError: bad operand type for unary ~: 'str'
```

when i split this command into two line, and all is fine

```
l = ~f'find -maxdepth 1 -name "*{x}*.zip" -type f'
l = l.splitlines()
```

By the way, ```data = (~'ls some_dir/').splitlines()```
this command is ok.
So maybe it only occurs on format string?

Thanks a lot

---
Ubuntu 20.04.2 LTS AArch64
Python 3.8.5
zxpy 1.2.1

**Comment by tusharsadhwani** [comment]
Hey, thanks for the bug report.

Yeah I didn't do extensive testing on that, this seems like a bug. But it's fixable, I'll get back to you ASAP.

**Comment by tusharsadhwani** [comment]
Okay, just fixed the bug and pushed v1.2.2 with the bugfix.

`pip install -U zxpy` and let me know if it works now.

**Comment by tjjh89017** [comment]
it works fine now.
Great thanks for this rapid fix!
